.aicoe-ci.yaml

```
@@ -0,0 +1,8 @@
+check:
+  - thoth-build
```

just noticed that this repo has `.pre-commit-config.yam`.
please use thoth-precommit check, it would do the pre-commit check on each pr.
```suggestion
  - thoth-build
  - thoth-precommit
```